nuxt.config.ts

```
@@ -33,4 +40,4 @@ localsConvention: "camelCase"
   // ],
 
   compatibilityDate: '2024-11-01',
-})
+})
```

ESLint

[error] Newline required at end of file but not found.

(style/eol-last)